sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/WebAppBaseImpl.java

```diff
@@ -1835,6 +1835,29 @@ public FluentImplT withoutIpAddressRangeAccess(String ipAddressCidr) {
         return (FluentImplT) this;
     }
 
+    @Override
+    @SuppressWarnings("unchecked")
+    public FluentImplT enablePublicNetworkAccess() {
+        this.ensureIpSecurityRestrictions();
```

Comment:
Why we need to call `ensureIpSecurityRestrictions()`? Anything here related to IP?